lib/runner.js

```diff
@@ -20,6 +23,7 @@ var chainableMethods = {
 		before: {type: 'before'},
 		after: {type: 'after'},
 		skip: {skipped: true},
+		todo: {skipped: true, todo: true},
```

Comment:
Nah there's no clear consensus either way. It's fine as it is, just different trade-offs in different areas.